server/data-migrations/1630003364070-TurnOnSSLForEveryPostgresDataSource.ts

```diff
@@ -1,22 +1,18 @@
 import { MigrationInterface, QueryRunner } from 'typeorm';
-import { DataSource } from '../src/entities/data_source.entity';
 
 export class TurnOnSSLForEveryPostgresDataSource1630003364070 implements MigrationInterface {
   public async up(queryRunner: QueryRunner): Promise<void> {
     const entityManager = queryRunner.manager;
-    const queryBuilder = queryRunner.connection.createQueryBuilder();
-    const dataSourceRepository = entityManager.getRepository(DataSource);
 
-    const dataSources = await dataSourceRepository.find({
-      where: { kind: 'postgresql' },
-      select: ['id', 'options'],
-    });
+    const dataSources = await entityManager.query('select id, options from data_sources where kind = $1', [
```

Comment:
Curious, what broke here to explicitly specify the SQL query?

Reply:
`options` is removed from DataSources entity